src/test/java/com/github/dockerjava/core/util/CertificateUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.dockerjava.core.util;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CertificateUtilsTest {
```

Comment:
ok, thanks. I created now folders for all mutations but getting the folder structure like you want dynamically get the path setup is not working yet.
I'm seeing other tests doing things like new File(Thread.currentThread().getContextClassLoader().getResource("testAddMultipleFiles") but that also doesn't look that clean to me. And in my case the code would also need to assume what the base dir is, which is also the case in the way I used before.